### PR TITLE
Bug 1635488 - Add mobile treescript workers

### PR DESCRIPTION
**Pull request by JohanLorenzo**
Modeled after https://github.com/mozilla-releng/k8s-autoscale/pull/96

### CODE REVIEW

File: configs/relengworker-prod/config.yml
Comment (MihaiTabara) [inline configs/relengworker-prod/config.yml:726]: I'm curious as to why this is set to `3` since we're expecting only Fenix releases to use it for now. Certainly not blocking.

File: configs/relengworker-prod/config.yml
Comment (MihaiTabara) [inline configs/relengworker-prod/config.yml:727]: I think it's good that this is bumped. Unclear how long the `git clone` operation will take on the scriptworker side so it's good to guard ourselves here 👍

**Review by MihaiTabara — APPROVED** [review]
Just to confirm, we can't merge this until we have cloudops-infra done and the workers show-up in Dockerhub. So this PR needs to be landed the last, once all the pieces are landed.

File: configs/relengworker-prod/config.yml
Comment (JohanLorenzo) [reply configs/relengworker-prod/config.yml:726]: I just copied what we had in gecko-tree-3, but I agree: 1 is good for now.

File: configs/relengworker-prod/config.yml
Comment (JohanLorenzo) [reply configs/relengworker-prod/config.yml:727]: I just copied what we had for `gecko-tree-3`. I guess the fenix repo is much simpler than `mozilla-release` so 300 should be good.

**Comment by MihaiTabara** [comment]
Merging this to `master` deploys the code to `dev`. I think we can fix the current brekage there by:
* need to deploy the treescript beforehand to renable the new workers in Jenkins and then rerun this
* revert the PR and merge again after the Jenkins will have been created.

I'm on with the first one since I was going to do it anyway 👍

**Pull request merged**
